docs/api.md

```diff
@@ -80,11 +85,14 @@ apiVersion: ipam.metal3.io/v1alpha1
 kind: IPAddress
 metadata:
   name: pool1-192-168-0-13
+  namespace: default
 spec:
   pool:
-    Name: pool1
+    name: pool1
+    namespace: default
   claim:
-    Name: claim1
+    name: claim1
```

Comment:
you can probably update this name here as I mentioned in https://github.com/metal3-io/ip-address-manager/pull/41#discussion_r678994631

Reply:
This is taken care in #41

Reply:
Sorry, disregard above comment. I updated document in this PR and 41 is only for examples folder related changes.